cmd/jaeger/badger_config.yaml

```diff
@@ -0,0 +1,34 @@
+service:
+  extensions: [jaeger_storage]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]
+
+extensions:
+  jaeger_query:
+    trace_storage: memstore
+    ui_config: ./config-ui.json
```

Comment:
this needs to be the same as in cmd/jaeger/config.yaml

Reply:
done!